guides/keras_cv/simsiam-with-kerascv.py

```diff
@@ -0,0 +1,636 @@
+"""
+Title: SimSiam Training with TensorFlow Similarity and KerasCV
+Author: [lukewood](https://lukewood.xyz), Ian Stenbit, Owen Vallis
+Date created: 2022/09/21
+Last modified: 2022/09/21
+Description: Train a KerasCV model using unlabelled data with SimSiam.
```

Comment:
Maybe this should s/model/backbone?

Reply:
most people still call any classifier a model - lets actually keep it.  Backbone is a colloquial term we use as a team for this class of models, but I wouldn't say its widely used yet.